data.json

```diff
@@ -994,5 +994,11 @@
     "githubUsername": "bevennyamande",
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
-  }
+  },
+  {
+    "name": "Kamlesh Joshi",
+    "githubUsername": "kamleshjoshi8102",
```

Comment:
Why do you want to add someone else?

Reply:
Actually, That person committed the changes to my forked Repo😅.
So I thought to create a PR for it.

Reply:
Can I do it?
Or Should I ask him to create a PR on this Repo?

Reply:
> Can I do it?
> Or Should I ask him to create a PR on this Repo?

Tough question I think its OK when you open a pr for this.